Source/Widgets/InfoBox/InfoBoxDescription.css

```diff
@@ -0,0 +1,42 @@
+body {
+    margin: 0;
+    padding: 0;
+}
+
+.cesium-infoBox-description {
+    font-family: sans-serif;
+    font-size: 13px;
+    padding: 4px 10px;
+    margin-right: 4px;
+    color: #edffff;
+}
+
+.cesium-infoBox-description table {
+    color: #edffff;
```

Comment:
Perhaps the fix could be to simple.czml itself, it has the wrong table class name.  But still, if someone puts an unstyled table in their description, it could end up written with black text.  Chrome is supplying a default text color for table elements.